pydantic/fields.py

```diff
@@ -325,8 +331,13 @@ def _validate_json(self, v: Any, loc: Tuple[str, ...]) -> Tuple[Optional[Any], O
         except (ValueError, TypeError) as exc:
             return v, ErrorWrapper(exc, loc=loc)
 
-    def _validate_sequence_like(
-        self, v: Any, values: Dict[str, Any], loc: 'LocType', cls: Optional['ModelOrDc']
+    def _validate_sequence_like(  # noqa: C901 (ignore complexity)
+        # Large if-else blocks are necessary to enable Cython optimization.
```

Comment:
comment probably best moved to line 336 or docstring.

Reply:
Done.